services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/HankeFactory.kt

```diff
@@ -119,6 +133,41 @@ class HankeFactory(
                 hankeStatus,
             )
 
+        fun createEntity(mockId: Int = 1): HankeEntity =
+            HankeEntity(
+                    id = mockId,
+                    status = DRAFT,
+                    hankeTunnus = defaultHankeTunnus,
+                    nimi = defaultNimi,
+                    kuvaus = defaultKuvaus,
+                    vaihe = SUUNNITTELU,
+                    suunnitteluVaihe = RAKENNUS_TAI_TOTEUTUS,
+                    onYKTHanke = true,
+                    version = 0,
+                    createdByUserId = defaultUser,
+                    createdAt = LocalDateTime.now().minusHours(1),
+                    modifiedByUserId = defaultUser,
+                    modifiedAt = LocalDateTime.now(),
```

Comment:
Using `now()` is effectively using a random value and I wouldn't recommend it.

Reply:
Uses datefactory now to limit randomness.